lookatme/parser.py

```diff
@@ -172,3 +183,21 @@ def parse_meta(self, input_data):
         yaml_data = "\n".join(yaml_data)
         data = MetaSchema().loads(yaml_data)
         return new_input, data
+
+    def _create_slides(self, tokens, md, number):
+        """Iterate on tokens and create slides out of them. Can create multiple
+        slides if the tokens contain progressive slide delimiters.
```

Comment:
Aha, I like how the slide delimiters create multiple slides instead of trying to modify an existing slide. This is great!